src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchain.cs

```diff
@@ -231,6 +232,8 @@ private Task InitBlockchain()
             return Task.CompletedTask;
         }
 
+        protected virtual IComparer<Transaction> CreateTxPoolTxComparer() => TxPool.TxPool.DefaultTxPoolComparer.Instance;
```

Comment:
DefaultTxPoolComparer should be renamed to GasPriceBasedTxComparer (nonce is always obvious)

Reply:
maybe GasBasedTxComparer ?